.github/workflows/ci.yaml

```
@@ -1,7 +1,15 @@
 name: CI
 
-on: [push, pull_request]
-
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.**"
```

```suggestion
      - "*"
```

as tags are currently not using a `v.` prefix..